tests/mock.py

```diff
@@ -0,0 +1,6 @@
+from unittest import mock
```

Comment:
Can I ask what this file is used for?

Reply:
https://medium.com/@piraveenaparalogarajah/what-is-mocking-in-testing-d4b0f2dbe20a